### PR TITLE
How to create a responsive React navigation bar using PrimeReact

### LINKED ISSUE #3847: How to create a responsive React navigation bar using PrimeReact
**Proposed title of article**

   How to create a responsive React navigation bar using PrimeReact.

**Introduction paragraph:**

  React.js is one of the most common JavaScript library used for building client-side rendering websites. This allows developers to make their website entirely rendered in the browser with JavaScript. 

   In this article, we will create a responsive React navigation bar using PrimeReact, which is a UI component library for React.js.

**Key takeaways:**
 
1.  What is PrimeReact.
2.  How to add PrimeReact library to react.js project.
3.  How to import and use PrimeReact components.
4.  How to make our navigation bar responsive using PrimeReact.

**References:**
N/A

### PR DESCRIPTION
**Pull request by Geraldoeze**
#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

#### Topic suggestion this closes

This closes #3847

### CODE REVIEW

**Comment by zolomohan** [comment]
Preliminary Revision done ✔️

**Comment by zolomohan** [comment]
@section-engineering-education/content-moderator-team Plagiarism Check already done in Old PR. https://github.com/section-engineering-education/engineering-education/pull/4236

**Review by zolomohan — APPROVED** [review]

File: content/articles/using-prime-react-in-react-applications/index.md [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+
+Within the div tag of our `app.js` file, we add our Navigation element:
+
+```javascript
+<div classname="App"> 
+ <Navigation />
+</div>. 
+```
+
+CSS styles can be added to change the pre-defined styles.
```
Comment (zolomohan) [inline content/articles/using-prime-react-in-react-applications/index.md:166]: Please add a screenshot of how this looks.

**Review by zolomohan — CHANGES_REQUESTED** [review]

**Comment by zolomohan** [comment]
@Geraldoeze It seems like most of the stuff you've covered here is covered in the documentation of the library. Is there anything you can add to make your article unique/original?

**Comment by WanjaMIKE** [comment]
@zolomohan does the update improve the quality? If not, we can close it and the author can do the revision outside the queue.

**Comment by zolomohan** [comment]
@WanjaMIKE We can close this article for now. This article requires more work. 
At this stage, it's only covering how to use the components which are already covered in the documentation of PrimeReact.

@Geraldoeze You can work on the updates to this article outside the queue. This does not mean this article will not be published. At this stage, this article does not provide a lot of value. Please improve upon this article or consider writing another topic.

Ping me if you create a new PR.

Cheers

**Pull request closed without merging**